projects/demo/src/modules/experimental/fade/fade.template.html

```diff
@@ -0,0 +1,113 @@
+<tui-doc-page
+    header="Fade"
+    package="EXPERIMENTAL"
+    type="components"
+>
+    <ng-template pageTab>
+        <tui-notification status="warning">
+            This code is
+            <strong>experimental</strong>
+            and is a subject to change. Expect final solution to be shipped in the next major version
+        </tui-notification>
+
+        <p>Wrapper that uses masking to fade out overflown content</p>
+
+        <tui-doc-example
+            id="basic"
+            heading="Basic"
+            [content]="example1"
+        >
+            <tui-fade-example-1></tui-fade-example-1>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="multiline"
+            heading="Multiline"
+            [content]="example2"
+        >
+            <tui-fade-example-2></tui-fade-example-2>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="vertical"
+            heading="Vertical"
+            [content]="example3"
+        >
+            <tui-fade-example-3></tui-fade-example-3>
+        </tui-doc-example>
+    </ng-template>
+
+    <ng-template pageTab>
+        <tui-doc-demo>
+            <tui-fade
+                class="fade"
+                [lineHeight]="lineHeight"
+                [offset]="offset"
+                [size]="size"
+            >
```

Comment:
`[orientation]`?

Reply:
It doesn't make sense on the API tab because you cannot change the content of it, so I only listed it in the API without being able to change it.